lib/AST/ASTImporter.cpp

```diff
@@ -2542,6 +2554,7 @@ Decl *ASTNodeImporter::VisitFunctionDecl(FunctionDecl *D) {
   }
 
   // Import the describing template function, if any.
+  // FIXME: This is omitted in upstream code. Do we actually need this?
```

Comment:
@szepet , IIRC you were dealing with template functions. What do you think about this FIXME?